doc/md/ci.mdx

```diff
@@ -94,3 +92,166 @@ fi
 
 </TabItem>
 </Tabs>
+
+## Lint migration files
+
+Changes to your project's Ent schema almost always result in a modification
+of your database. If you are using [Versioned Migrations](/docs/versioned-migrations)
+to manage changes to your database schema, we recommend running [migration linting](https://atlasgo.io/versioned/lint)
+as part of your continuous integration flow. This is done for multiple reasons:
+
+* Linting replays your migration directory on a real database to make sure all SQL statements
+  are valid and in the correct order.
+* Directory integrity is enforced - ensuring that history wasn't accidentally changed and
+  that migrations that are planned in parallel are unified to a clean linear history.
+* Destructive changes are detected notifying you of any potential data loss that may be
+  caused by your migrations way before they reach your production database.
+* Linting detects data-dependant changes that _may_ fail upon deployment and require
+  more careful review from your side.
+
+If you're using GitHub, we recommend using the [Official Atlas Action](https://github.com/ariga/atlas-action)
+to run migration linting during CI.
+
+Add `.github/workflows/atlas-ci.yaml` to your repo with the following contents:
+<Tabs
+defaultValue="mysql"
+values={[
+    {label: 'MySQL', value: 'mysql'},
+    {label: 'MariaDB', value: 'maria'},
+    {label: 'PostgreSQL', value: 'postgres'},
+]}>
+<TabItem value="mysql">
+
+```yaml
+name: Atlas CI
+on:
+  # Run whenever code is changed in the master branch,
+  # change this to your root branch.
+  push:
+    branches:
+      - master
+  # Run on PRs where something changed under the `ent/migrate/migrations/` directory.
+  pull_request:
+    paths:
+      - 'ent/migrate/migrations/*'
+jobs:
+  lint:
+    services:
+      # Spin up a mysql:8.0.29 container to be used as the dev-database for analysis.
+      mysql:
```

Comment:
This service is called `mysql` but maria and postgres are called with their version `postgres10` / `maria107`